README.org

```diff
@@ -141,7 +146,13 @@
                       :major-modes '(python-mode)
                       :server-id 'pyls))
   #+END_SRC
-
+** FAQ
+*** How to configure a server with local variables?
+    Add ~lsp~ server call to ~hack-local-variables-hook~ which runs right after the local variables are loaded.
+    #+BEGIN_SRC emacs-lisp
+      (add-hook 'hack-local-variables-hook
+                (lambda () (when (derived-mode-p 'prog-mode) (lsp))))
```

Comment:
The `prog-mode` example should be specific as I have mentioned that `prog-mode` might not be a good example...

Reply:
see above.